src/components/Accordion.js

```diff
@@ -36,7 +36,7 @@ const Accordion = React.createClass({
   renderContainer: function(item, i) {
     const { opened, created } = this.state;
     const containerClassName =
-          item.header.toLowerCase().replace(/\s/g, "-") + "-pane";
+      `${item.header.toLowerCase().replace(/\s/g, "-")}-pane`;
```

Comment:
Actually I agree. There is something about "more than one period" in a template that makes me feel like it's wrong.